devicetypes/smartthings/zwave-basic-window-shade.src/zwave-basic-window-shade.groovy

```diff
@@ -58,11 +58,25 @@ metadata {
 				defaultValue: false,
 				displayDuringSetup: false
 			)
+            		
+			input(title: "Aeotec Nano Shutter settings",
+			    description: "Sets the Open/Close time.",
+			    displayDuringSetup: false,
+			    type: "paragraph",
+			    element: "paragraph")
+
+			input("calibrationTime", "number",
+			    title: "Set the open/close time for motor, range = 5 - 255",
```

Comment:
you could add a `range: "5..255"` field instead of this text

Reply:
@greens - fixed